### ISSUE TITLE
Impossible to connect second part of #2

### ISSUE THREAD

**Issue by ernesst** [bhdouglass/owncloud-files-scope#3]
I still cannot connect,

See the log 
```
ownCloud-files-scope.bhdouglass LOG: starting (owncloud-files-scope.bhdouglass_owncloud-files)
ownCloud-files-scope.bhdouglass LOG: running
[2016-03-01 07:54:47.359605] INFO: Registry: RegistryObject::ScopeProcess::exec(): Process for scope: "owncloud-files-scope.bhdouglass_owncloud-files" started
https://YYYYYY:XXXXXX@ZZZZZ.ZZZZ.zzzz/remote.php/webdav/
ownCloud-files-scope.bhdouglass LOG: search: 
ownCloud-files-scope.bhdouglass ERROR: error reading dir: You do not have access to this service

```
Regards

**Comment by bhdouglass** [comment]
Are you using your password with special characters?

**Comment by ernesst** [comment]
Yes i'm using special characters

**Comment by bhdouglass** [comment]
Which ones in particular? Then I can use them to test.

**Comment by bhdouglass** [comment]
I believe this to be a problem in either the JS scopes code or the scope implementation. I've created this bug to hopefully get to the bottom of this - https://bugs.launchpad.net/unity-js-scopes/+bug/1552082

**Comment by ernesst** [comment]
Thanks I vote for it ^^

**Comment by bhdouglass** [comment]
Awesome, hopefully we can get to the bottom of this.

**Comment by bhdouglass** [comment]
The guys working on the scopes api have determined that this is a bug in the scope settings. Unfortunately this means we have to wait until an OTA update for this to be fixed.

**Comment by ernesst** [comment]
Thanks for the update,

**Comment by bhdouglass** [comment]
I've implemented a workaround that should allow you to use your special characters in your username/password.

**Comment by bhdouglass** [comment]
This is available in v1.7

**Comment by bhdouglass** [comment]
Just FYI: the bug I created for the scope implementation has just been marked as "fix committed".
